### ISSUE TITLE
Extend scope with start and offset

### ISSUE THREAD

**Issue by thhermansen** [CompanyBook/massive_record#54]
So we can do a

```
User.start("id-have-starts-with-this").offset("start-from-here").limit(10).each do |record|
  record.do_something
end
```

**Comment by thhermansen** [comment]
By the way. I do think the start and the offset might be a bit confusing. Start is not explicitly stating that the string-id have to start with given value. Maybe we are gonna rename these.

**Comment by thhermansen** [comment]
Fixed in: 1549b6d53e364cb13b5b8bd4b3d84ea9653c091d
